BUILD

```diff
@@ -13,4 +13,5 @@ python_test_utils(name="test_utils")
 _docker_environment(
     name="docker_env",
     image="python:3.9",
+    python_bootstrap_search_path=["<PATH>"],
```

Comment:
This is to support the case where `[python-bootstrap].search_path` is being overridden by a Pants developer to point at `<PYENV>` instead: this overrides it back to something which is valid for this image.

Reply:
@stuhood Note that for python bootstrapping, keeping the default value of `["<PYENV>", "<PATH>"]` is acceptable, the `<PYENV>` will be discarded, leaving only `<PATH>`.

`<PYENV>` in a manually-specified value is not acceptable.

Reply:
> `<PYENV>` in a manually-specified value is not acceptable.

It is what I use by default via a `.pants.rc`, in order to avoid ever using a system interpreter.

Reply:
ack.

Obviously in favour of making the `_docker_environment` config explicitly valid here. I'm just suggesting highlighting this as not strictly necessary given the config in Pants' `pants.toml`.